specs/subresourceintegrity/spec.markdown

```diff
@@ -412,7 +412,7 @@ the user agent.
     [parsing <var>metadataList</var>][parse].
 2.  If <var>parsedMetadata</var> is `no metadata`, return `true`.
 3.  If [<var>response</var> is not eligible for integrity
```

Comment:
It refers to [this algorithm](https://w3c.github.io/webappsec/specs/subresourceintegrity/#is-varresponsevar-eligible-for-integrity-validation) which says it needs to be CORS or same-origin to be eligible.